### DISCUSSION TITLE
Kata6 "TypeError: 'list' object is not callable"

### DISCUSSION THREAD

**Discussion by GerardoSolanoo** [LaunchX-InnovaccionVirtual/CursoIntroPython#274] [Q&A] [answered]
Buenas tardees, recientemente he tenido algunos problemas a la hora de ejecutar la Kata 6 a que marca un error en las listas pero no encuentro el por que de el error
![1Captura](https://user-images.githubusercontent.com/76238126/154131992-57ad20d1-7b92-4c34-88ca-c430f8257999.PNG)

**Comment by ManuelHN** [comment]
Sugiero ejecutar las prácticas fuera del archivo md, en una propia notebook, porque me parece que python está tratando de buscar una referencia a la lista anterior, no tendrás problemas si lo haces en una notebook aparte.

**Comment by kriztobal** [comment]
La variable planets guarda una lista de valores, no se pueden mostrar esos valores solo con print directamente como si fuera una variable que no sea de tipo lista y que guarda solo un valor.

Además debes leer lo que solicita el print, si te fijas bien pide el número de planetas, no el nombre de ellos, debes buscar un método que haga que muestre la cantidad de planetas que hay en la lista en cuestión. Eso está en los apuntes previos a la kata.

**Comment by BriaAlba** [comment]
Te falto escribir una coma en planets. ya que no estas diferenciando entre la variable y el texto. solo coloca la coma y ya te dará el resultado de print.

**Comment by GerardoSolanoo** [comment, marked as answer]
Hola, ya se solucionó, estuve unas cuatro horas para darme cuenta que solo era cerrar y abrir el vsc, muchas gracias a todos por tomarse el tiempo de contestar mi duda.
